app/models/user.rb

```diff
@@ -2,6 +2,9 @@ class User < ActiveRecord::Base
   include Activatable
   include Linkable
 
+  #on registration track which project the user signs up with
+  attr_accessor :project_id
```

Comment:
nope, was just going to pass it to the welcome emailer to modify the content there. I'll add a field to the users table to track it.